identity-enabler/deviceId-mobile-app/src/pages/InvalidCredential.svelte

```diff
@@ -14,7 +14,7 @@
         if (delay < 0) {
             await playAudio("invalid");
         } else {
-            setTimeout(async () => await playAudio("invalid"), delay);
+            setTimeout(async () => playAudio("invalid"), delay);
```

Comment:
are you sure about this change?

Reply:
Yes, the anonymous function is defined as async which must return a promise, either implicitly using `await` or explicitly like here where the `playAudio` Promise is returned. These two are equivalent and in fact using `await` is redundant (I believe it would wrap the `playAudio` Promise in an extra Promise).